### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by elmar8287**
- Add RSpec as a gem to the project.
- Create a Request spec file for all controllers.
- Make sure that for each action you check:

1. If the response status was correct.
2. If a correct template was rendered.
3. If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      expect(response).to_not render_template('show')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include('Users#index')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include('Users#index')
+    end
```
Comment (Dipeshtwis) [inline spec/requests/users_spec.rb]: - [ ] There is a duplicate test case wriiten here and also on line 46.

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    it 'check status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders correct template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'renders correct template' do
+      expect(response).to_not render_template('show')
```
Comment (Dipeshtwis) [inline spec/requests/users_spec.rb:15]: - [ ] Please fix the test description. It should be `not render correct template`.

File: spec/requests/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    it 'check status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders correct template' do
+      expect(response.body).to render_template('show')
+    end
+
+    it 'renders correct template' do
+      expect(response).to_not render_template('index')
```
Comment (Dipeshtwis) [inline spec/requests/users_spec.rb:35]: - [ ] Please fix the test description. It should be `not render correct template`.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi Team, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

Check the comments under the review

- [x] Readme file should have command to test the project.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (elmar8287) [reply spec/requests/users_spec.rb]: Done

File: spec/requests/users_spec.rb (hunk 2 above)
Comment (elmar8287) [reply spec/requests/users_spec.rb:15]: Done

File: spec/requests/users_spec.rb (hunk 3 above)
Comment (elmar8287) [reply spec/requests/users_spec.rb:35]: Done

**Comment by elmar8287** [comment]
Thanks for the review. I have fixed your notes. 👍

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @elmar8287, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [x] As previously requested in the review summary, add the testing command to the readme file.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Thanks for the note 👍  I have fixed it 🥇

**Review by AshakaE — APPROVED** [review]
### Hi @elmar8287  🙋‍♀️, 

# STATUS APPROVED ✔

Thanks for making the changes 👏 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Thanks for the review! 👍

**Pull request merged**